src/components/content/MessageWithLink.tsx

```diff
@@ -15,9 +16,12 @@ function MessageWithLink({ messageID }: Props) {
             values={{
                 docLink: (
                     <ExternalLink
-                        link={intl.formatMessage({
-                            id: `${messageID}.docPath`,
-                        })}
+                        link={
+                            link ??
+                            intl.formatMessage({
+                                id: `${messageID}.docPath`,
+                            })
+                        }
```

Comment:
This kinda sucks but might be okay. We used to have a steady pattern on how all inline links were handled and now we are breaking it... but it feels better breaking it than it would to duplicate all the code in this component.